package.json

```diff
@@ -42,9 +42,16 @@
   "tree-sitter": [
     {
       "scope": "source.bash",
+      "injection-regex": "(shell|bash|zsh|sh)",
+      "first-line-regex": "^#!.*\\b(sh|bash|dash|zsh)\\b.*$",
       "file-types": [
         "sh",
-        "bash"
+        "bash",
+        "zsh",
+        ".bashrc",
+        ".bash_profile",
```

Comment:
what is file-types used for? does it really work with extensions and just filenames like this?

Reply:
It is matched by suffix according to https://tree-sitter.github.io/tree-sitter/syntax-highlighting#language-detection, so I could probably actually deduplicate some of these (and probably means that this was matching `zsh` already)

This is read by `tree-sitter-cli` to determine how to highlight a given file, but I think it also serves as a basis for what editors use. I pulled a subset of what Helix uses https://github.com/helix-editor/helix/blob/9ed3dc52e0d56f0f0fd721d7f10da5f935f56ef9/languages.toml#L795-L835